easybuild/easyconfigs/s/Spark/Spark-1.3.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'Tarball'
+
+name = 'Spark'
+version = '1.3.0'
+
+homepage = 'http://spark.apache.org'
+description = """Spark is Hadoop MapReduce done in memory"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['%(namelower)s-%(version)s-bin-hadoop2.3.tgz']
+source_urls = ['http://www.apache.org/dyn/closer.cgi/spark/%(namelower)s-%(version)s']
+
+java = 'Java'
+javaver = '1.7.0_40'
+
+dependencies = [(java, javaver)]
+
+sanity_check_paths = {
+    'files': ['bin/spark-shell'],
+    'dirs': ['lib', 'python']
+}
+
+modextrapaths = {'PYTHONPATH': 'python'}
```

Comment:
Pyspark is optional but if you want to use it, PYTHONPATHH needs to have this dir in it. Hadoop is also optional but I don't bind a version.